### PR TITLE
Added reparation recipe

### PR DESCRIPTION
**Pull request by RickGelhausen**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by RickGelhausen** [comment]
Hi, @bgruening . Is there a way to test a recipe locally before merging (after circleci stops throwing errors)?
The input files for the tool are too large to add, and I would like to test my changes to the tool before merging a potentially faulty recipe. Do you have any suggestion?

File: recipes/reparation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - positive_set.patch
+
+build:
+  number: 0
+  skip: True [osx or py35]
+
+requirements:
+  host:
+    - perl
+    - perl-posix
```
Comment (bgruening) [inline recipes/reparation/meta.yaml]: you do not need all the build requirements isn't it?

File: recipes/reparation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - glimmer
+    - blast
+
+test:
+  commands:
+    - reparation.pl --help
+
+about:
+  home: https://github.com/Biobix/REPARATION
+  license: GGPL3
```
Comment (bgruening) [inline recipes/reparation/meta.yaml]: GPL3

**Comment by bgruening** [comment]
@RickGelhausen you can compile the package locally with either `conda build` or `circleci build`.

File: recipes/reparation/meta.yaml (hunk 1 above)
Comment (RickGelhausen) [reply recipes/reparation/meta.yaml]: Yes thanks, my bad.

File: recipes/reparation/meta.yaml (hunk 2 above)
Comment (RickGelhausen) [reply recipes/reparation/meta.yaml]: Thanks

**Comment by RickGelhausen** [comment]
@bgruening Thank you, I am actually using `circleci build` locally. But I was not aware of `conda build`, I think that is what I was looking for.

**Comment by RickGelhausen** [comment]
Ok nice, I did not know about that. I will check it out. Thanks.

**Pull request closed without merging**